classes/webservice/WebserviceSpecificManagementImages.php

```diff
@@ -1032,11 +1027,12 @@ protected function writeImageOnDisk($base_path, $new_path, $dest_width = null, $
      */
     protected function writePostedImageOnDisk($reception_path, $dest_width = null, $dest_height = null, $image_types = null, $parent_path = null)
     {
+        $imgMaxUploadSize = ((int) Configuration::get('PS_LIMIT_UPLOAD_IMAGE_VALUE')) * 1000 * 1000;
```

Comment:
Seems ok to me, just one thing I'm a bit wary of, are we sure that the dropzone JS component computes the max size this way? Because it could be:

```suggestion
        $imgMaxUploadSize = ((int) Configuration::get('PS_LIMIT_UPLOAD_IMAGE_VALUE')) * 1024 * 1024;
```

which would result in slightly different behavior. But it's not blocking anyway the difference is minor. However, this makes me realize I'm not sure this limit has been implemented on the product page v2.

@NeOMakinG do you know if the component used for v2 has such a feature? We should also probably handle this in the PHP API side internally.

Reply:
https://docs.dropzone.dev/configuration/basics/configuration-options

`maxFilesize` is 256 bytes by default